db/schema.rb

```diff
@@ -10,8 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_11_30_050124) do
-
+ActiveRecord::Schema[7.0].define(version: 2022_03_29_203956) do
```

Comment:
Why do we need to update the schema to Rails 7? Is this required for this PR?

Reply:
To some degree yes. I recreated the migrations on the current master (which uses Rails 7). This created Rails 7.0 migrations which made these changes to the schema when db:migrate was ran. I do intend to open this as a separate PR in rubygems.org as it would be needed at some point.